### PR TITLE
Data retention - revised with edits

### PR DESCRIPTION
**Pull request by mmfred**
Added in edits from @kaileyhaynes

### CODE REVIEW

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:14]: "At a glance, get a the sources" -- is something missing between "a" and "the" or should "a" simply be deleted?

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:16]: we've talked about, it's a good idea to have a description in the "" part of the image (rather than just repeating the png filename, which is not particularly useful for customers). But style guide has not yet been updated to reflect that, so it's up to you whether to take a sec to fix. https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#labeling

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:346]: "might" or "can" ? I think this is a voice and tone choice for individual writers, not sure though.

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:348]: is there a difference between "edition" and "tier" ? I think I've seen both words used but not clear if they are interchangeable or mean very specific things.

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:350]: Is this still true, even with legacy accounts? I was under the impression they migrated everything off Insights, but not sure if that applies to subscriptions too.

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:359]: is this capitalization vs. lower case intentional? I'm not sure how it all looks anymore, sorry!

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:363]: This first sentence has 30+ words. I wonder if it would help to start a new sentence after "immediately disappear"? The second sentence is pretty long too. Maybe this is simply the author's writing style? As a non-user trying to understand the nuances, it helps me to read shorter sentences or bulleted lists. But that's just me. Your call!

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:373]: Is parent/child terminology still used? I thought they were switching to account/sub-account but I may be wrong.

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:375]: I like how this one does have a description in the "" section of the image!

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:378]: To avoid sentences ending with "to," sometimes I reword to something like "Select the child (sub?) accounts where  you want to copy retention values."

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:387]: I personally don't like "out-of-the-box" because literally, we don't sell New Relic software on disc, packaged in a box! :-) But that's just me. Maybe consider something like "default metrics" (to differentiate from your custom metrics" or "metrics automatically available in New Relic" or something like that?

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (barbnewrelic) [inline src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:457]: I love your edit here!!

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
This is a really great rewrite! I learned a lot! I do have some questions and suggestions, take 'em or leave 'em. Then, no need to request a second review from me. Feel free to use your admin powers to merge when you're ready.

File: src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx
Comment (zuluecho9) [reply src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx:373]: it's opposite: it was master/sub-account and now it's parent and child account. But in v2 world, these things are much less important and not exposed to v2 customers (they have no way of understanding what's a parent account or child account, it's primarily under the hood). Might be worth us talking about that, Michelle, as reading this, I'm not sure as a v2 user what this would mean. But feel free to roll it out, maybe just something for me to review later, and I'll make note of it.

**Comment by barbnewrelic** [comment]
good to know, thanks for setting me straight!

**Review by mmfred — COMMENTED** [review]
Thanks for the edits @barbnewrelic

**Pull request merged**
